packages/wp-now/src/download.ts

```diff
@@ -12,8 +12,19 @@ import {
 	WP_NOW_PATH,
 } from './constants';
 
+export function isValidWordpressVersion(version: string): boolean {
+	const versionPattern =
+		/^latest$|^(?:(\d+)\.(\d+)(?:\.(\d+))?)((?:-beta(?:\d+)?)|(?:-RC(?:\d+)?))?$/;
+	return versionPattern.test(version);
+}
+
 function getWordPressVersionUrl(version = DEFAULT_WORDPRESS_VERSION) {
-  return `https://wordpress.org/wordpress-${version}.zip`;
+	if (!isValidWordpressVersion(version)) {
```

Comment:
This is repeated in quite a few places, let’s add a canonical `@wp-playground/wordpress` package and import this logic from there

https://github.com/search?q=repo%3AWordPress%2Fwordpress-playground%20supportedWordPressVersions&type=code

Reply:
a good follow-up issue? seems like something that would require a lot of changes unrelated to this PR…